### DISCUSSION TITLE
Please help me, Thanks a lot.

### DISCUSSION THREAD

**Discussion by kailintian** [ruffle-rs/ruffle#2955] [Q&A] [answered]
I read this :https://github.com/ruffle-rs/ruffle/wiki/Using-Ruffle, I try to use ruffle on safari, but I don't know what to enter in terminal. 
Last login: Fri Jan 29 15:29:41 on ttys000
apple@AppledeMacBook-Pro ~ % cd/Users/apple/Downloads/ruffle_nightly_2021_01_29_extension
zsh: no such file or directory: cd/Users/apple/Downloads/ruffle_nightly_2021_01_29_extension
apple@AppledeMacBook-Pro ~ % cd /Users/apple/Downloads/ruffle_nightly_2021_01_29_extension
apple@AppledeMacBook-Pro ruffle_nightly_2021_01_29_extension % xcrun safari-web-extension-converter
Please provide the path to the extension to convert.
apple@AppledeMacBook-Pro ruffle_nightly_2021_01_29_extension % xcrun
Usage: xcrun [options] <tool name> ... arguments ...

Find and execute the named command line tool from the active developer
directory.

The active developer directory can be set using `xcode-select`, or via the
DEVELOPER_DIR environment variable. See the xcrun and xcode-select manual
pages for more information.

Options:
  -h, --help                  show this help message and exit
  --version                   show the xcrun version
  -v, --verbose               show verbose logging output
  --sdk <sdk name>            find the tool for the given SDK name
  --toolchain <name>          find the tool for the given toolchain
  -l, --log                   show commands to be executed (with --run)
  -f, --find                  only find and print the tool path
  -r, --run                   find and execute the tool (the default behavior)
  -n, --no-cache              do not use the lookup cache
  -k, --kill-cache            invalidate all existing cache entries
  --show-sdk-path             show selected SDK install path
  --show-sdk-version          show selected SDK version
  --show-sdk-build-version    show selected SDK build version
  --show-sdk-platform-path    show selected SDK platform path
  --show-sdk-platform-version show selected SDK platform version
apple@AppledeMacBook-Pro ruffle_nightly_2021_01_29_extension %

**Comment by IAmYhvr** [comment, marked as answer]
It looks like you forgot the `.` at the end of the command when running
```sh
xcrun safari-web-extension-converter .
```
in the fourth step.

**Reply by kailintian** [reply]
Thanks a lot. It is solved now.
